integration-cli/docker_utils.go

```diff
@@ -389,21 +371,16 @@ func deleteAllImages() error {
 		fields := strings.Fields(l)
 		imgTag := fields[0] + ":" + fields[1]
 		if _, ok := protectedImages[imgTag]; !ok {
-			if fields[0] == "<none>" {
+			if fields[0] == "<none>" || fields[1] == "<none>" {
```

Comment:
This make sure it will also delete image like `imageName:<none>` which are generated on some `DockerRegistrySuite` tests. There was some left overs, and as we ignored the errors, we didn't even know it…